File: component/component-mirai/src/test/java/love/forte/test/MyListener.java
package love.forte.test;

import love.forte.common.ioc.annotation.Beans;
import love.forte.simbot.annotation.OnPrivate;
import love.forte.simbot.api.message.events.PrivateMsg;
import love.forte.simbot.api.sender.Sender;

/**
 * @author ForteScarlet
 */
@Beans
public class MyListener {

    @OnPrivate
    public void reply(PrivateMsg msg, Sender sender) {
        sender.sendPrivateMsgAsync(msg, msg.getMsgContent());
    }


}

File: boots/simboot-core/src/test/kotlin/MyListener.kt
import love.forte.di.annotation.Beans
import love.forte.simboot.annotation.Listener
import love.forte.simbot.event.MessageEvent

/**
 *
 * @author ForteScarlet
 */
@Beans
class MyListener {

    @Listener
    fun MessageEvent.listener() {

    }

}